### PR TITLE
Favorite tests

### PR DESCRIPTION
**Pull request by Stearnzy**
Description:  
* Tests for favorites written

* [x] Lint code

### CODE REVIEW

**Review by Callbritton — APPROVED** [review]
Thank you for including these additional tests and for making the existing tests more robust

File: spec/features/dashboard/dashboard_spec.rb [hunk 1]
```diff
@@ -31,38 +32,40 @@
       expect(page).to have_link('Electricity')
     end
 
-    xit "should redirect them if the user already has a saved location" do
+    xit 'should redirect them if the user already has a saved location' do
```
Comment (jakeheft) [inline spec/features/dashboard/dashboard_spec.rb:55]: Should this still be x'ed out?

File: spec/features/dashboard/dashboard_spec.rb [hunk 2]
```diff
@@ -31,38 +32,40 @@
...
+        .to_return(status: 200, body: json_response, headers: {})
 
-       visit '/dashboard'
+      visit '/dashboard'
 
-       expect(current_path).to eq("/address")
+      expect(current_path).to eq('/address')
     end
 
     xit 'not search with an invalid zipcode' do
```
Comment (jakeheft) [inline spec/features/dashboard/dashboard_spec.rb:76]: Same^^?

**Review by jakeheft — APPROVED** [review]
Great work getting webmock to work!

**Pull request closed without merging**